hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseOptions.java

```diff
@@ -141,25 +141,25 @@ public static synchronized HbaseOptions instance() {
 
     public static final ConfigOption<Integer> HBASE_VERTEX_TABLE_REGION_NUM =
             new ConfigOption<>(
-                    "hbase.hbase_vertex_table_region_num",
+                    "hbase.vertex_table_region_num",
```

Comment:
can we also rename to hbase.edge_partitions and update HBASE_VERTEX_TABLE_REGION_NUM and option description?